bigbluebutton-html5/imports/ui/components/whiteboard/cursors/component.jsx

```diff
@@ -53,7 +53,15 @@ const Cursors = (props) => {
 
   const [panGrabbing, setPanGrabbing] = React.useState(false);
 
-  const start = () => {
+  const start = (event) => {
+    const targetElement = event.target;
```

Comment:
```suggestion
    const targetElement = event?.target;
```

if you're checking a few lines under whether `event` is defined, might as well check here too